app/helpers/pages_helper.rb

```diff
@@ -9,6 +9,10 @@ def page_nav_item(text, path, strict=true)
     end
   end
 
+  def serialize(data, field)
+    (data.stringify_keys[field.to_s] || {}).to_json.html_safe
+  end
```

Comment:
No I mean when the value is empty. 
```
irb(main):001:0> h = {a: 'b'}
=> {:a=>"b"}
irb(main):002:0> h = {a: ''}
=> {:a=>""}
irb(main):003:0> h.fetch(:a, 'lol')
=> ""```
That case matters because these are getting injected into JS, so it would cause a syntax error.